pkg/tfbridge/walk.go

```diff
@@ -168,3 +265,347 @@ func LookupSchemaInfoPath(schemaPath SchemaPath, schemaInfo *SchemaInfo) *Schema
 		return nil
 	}
 }
+
+// Drill down a path from a SchemaInfo object and find a matching SchemaInfo. If no
+// SchemaInfo exists, it will be created.
+func getSchemaInfoPath(schemaPath SchemaPath, schemaInfo *SchemaInfo) *SchemaInfo {
```

Comment:
https://github.com/pulumi/pulumi-terraform-bridge/blob/master/pkg/tfbridge/walk.go#L131 is this materially different from LookupSchemaInfoMapPath, maybe null handling? 

"If no SchemaInfo exists, it will be created."

Looks like that's the difference. Perhaps call it "getOrCreateSchemaInfoByPath"